docs/extensions/dnsrecord.md

```diff
@@ -127,10 +127,20 @@ If this feature gate is enabled, all three DNS records mentioned above (internal
 These providers can be used for `DNSEntry` resources needed by workloads deployed on the shoot cluster.
 
 If the feature gate is disabled, Gardener will not create any `DNSRecord` resources and use `DNSProvider` / `DNSEntry` resources for its DNS records. 
-Currently, the feature gate is still in `Alpha` stage and is therefore disabled by default.
+The feature gate is enabled by default starting from `v1.39`.
 
 In order to successfully reconcile a shoot with the feature gate enabled, extension controllers for `DNSRecord` resources for types used in the default, internal and custom domain secrets should be registered via `ControllerRegistration` resources.
 
+### List of first versions of extensions that supports `DNSRecord`
+
+| Extension                                    | Version  |
+| -------------------------------------------- | -------- |
```

Comment:
What about other provider extensions like provider-metal, provider-vsphere and provider-kubevirt? Even if they don't have support for DNSRecords, does it make sense to add them to the table and make it clear somehow that these provider extensions don't have support for DNSRecords currently?

Reply:
I can do that, yes. Makes sense. 
Shall I put a `N/A` on the version column?

Reply:
Yes, `N/A` is fine.